studio/components/interfaces/Auth/RedirectUrls/RedirectUrls.tsx

```diff
@@ -1,20 +1,26 @@
-import { useState } from 'react'
-import { object, string } from 'yup'
-import { observer } from 'mobx-react-lite'
 import * as Tooltip from '@radix-ui/react-tooltip'
-import { Button, Form, Input, Modal } from 'ui'
 import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { observer } from 'mobx-react-lite'
+import { useState } from 'react'
+import { Button, Form, Input, Modal } from 'ui'
+import { object, string } from 'yup'
 
-import { useCheckPermissions, useStore } from 'hooks'
+import { useParams } from 'common'
 import { FormHeader } from 'components/ui/Forms'
+import { useAuthConfigQuery } from 'data/auth/auth-config-query'
+import { useAuthConfigUpdateMutation } from 'data/auth/auth-config-update-mutation'
+import { useCheckPermissions, useStore } from 'hooks'
 import { urlRegex } from '../Auth.constants'
 import RedirectUrlList from './RedirectUrlList'
 
 const RedirectUrls = () => {
-  const { authConfig, ui } = useStore()
+  const { ui } = useStore()
+  const { ref: projectRef } = useParams()
+  const { data: config } = useAuthConfigQuery({ projectRef })
+  const { mutate: updateAuthConfig } = useAuthConfigUpdateMutation()
```

Comment:
let's handle the loading and error states here

Reply:
Done.